website/content/en/status/report-2023-01-2023-03/cloud-init.adoc

```diff
@@ -0,0 +1,27 @@
+=== FreeBSD as a Tier 1 cloud-init Platform
+
+Links: +
+link:https://cloud-init.io/[cloud-init Website] URL: link:https://cloud-init.io/[https://cloud-init.io/] +
+link:https://cloudinit.readthedocs.io/en/latest/[cloud-init Documentation] URL: link:https://cloudinit.readthedocs.io/en/latest/[https://cloudinit.readthedocs.io/en/latest/] +
+link:https://github.com/canonical/cloud-init/blob/main/WIP-ONGOING-REFACTORIZATION.rst[cloud-init ongoing refactorization] URL: link:https://github.com/canonical/cloud-init/blob/main/WIP-ONGOING-REFACTORIZATION.rst[https://github.com/canonical/cloud-init/blob/main/WIP-ONGOING-REFACTORIZATION.rst]
+
+Contact: Mina Galić <FreeBSD@igalic.co>
```

Comment:
Please note that in your other report you used lower cases in your email address (`freebsd@igalic.co`), which is also what I used in https://github.com/freebsd/freebsd-doc/commit/2b391054846d92eafbab31225526b09aa5e4a041 and what you use as a ports maintainer according to [portscout](https://portscout.freebsd.org/freebsd@igalic.co.html).

While we usually prefer the `FreeBSD` capitalization, since this is your personal email address and it is already well established in the lower case variant, I suggest you keep it lower case. Anyway, it would be better to have consistency at least between your two reports.